open-codegen/opengen/builder/optimizer_builder.py

```diff
@@ -464,8 +464,9 @@ def __build_python_bindings(self):
                                   'win32': ('.dll', '.pyd')}
         (original_lib_extension,
          target_lib_extension) = pltform_extension_dict[sys.platform]
+        optimizer_prefix = "lib" if sys.platform != "win32" else ""
```

Comment:
Looks good. Have you tested this on Windows? It would be convenient to have a proper Windows CI. [Appveyor](https://ci.appveyor.com/project/alphaville/optimization-engine/branch/master) tests the Rust part of the code, but not opengen. Do all tests pass on your system?